src/gui/plugins/component_inspector/ExpandingTypeHeader.qml

```diff
@@ -46,10 +49,12 @@ Rectangle {
       fillMode: Image.Pad
       Layout.alignment : Qt.AlignVCenter
       source: content.show ?
-          "qrc:/Gazebo/images/minus.png" : "qrc:/Gazebo/images/plus.png"
+          "qrc:/Gazebo/images/chevron-down.svg" : "qrc:/Gazebo/images/chevron-right.svg"
     }
     TypeHeader {
       id: typeHeader
+      headerText: expandingHeaderText
+      headerToolTip: expandingHeaderToolTip
```

Comment:
I don't see tooltips for the expanding header:

![pose_header](https://user-images.githubusercontent.com/5751272/144679903-d925961d-6301-4ad1-a4db-f242e570742a.gif)